app/assets/stylesheets/style.scss

```diff
@@ -211,13 +219,35 @@ p.list-group-item-text {
 column-count: 3;
 }
 
-.footer {margin: 4em 0 9em 0;
-text-align:center;}
+/* Footer */
 
-.footer img{
+.footer {
+  margin: 4em 0 9em 0;
+  text-align:center;
+}
+
+.footernav {
+  margin: 1em 0 3em 0;
+  padding: 0;
+  list-style: none;
+}
+
+.footernav li {
```

Comment:
I would nest the `li` inside `.footernav` instead of putting it below like this.